tox.ini

```diff
@@ -28,6 +28,7 @@ extras =
     eventlet: eventlet
     sasl: sasl
 deps =
+    pytest-timeout
```

Comment:
I see both pytest and pytest-cov are in the `constraints.txt` file at the root level of the repo, so probably it'd be good to add this there as well for consistency.